webpack-file/src/component.js

```diff
@@ -1,3 +1,5 @@
+import Storage from './storage.js';
+
```

Comment:
[OPTIONAL] It's best to keep these files in the root directory:

![image](https://user-images.githubusercontent.com/38088818/194704283-e39f1a65-1115-4a3b-8919-c4138bcbbbcb.png)

- It's always a great idea to keep config files like the `package.json`, `.gitignore`, `webpack.config.js`, `node_modules`, etc. in the root directory since most of these files are necessary to start the app.   This is to avoid `path` issues and having to navigate through directories to start the app.
- I highly recommend that you move all the files inside the `webpack-file` directory back to the main directory and delete it.